zkevm/src/circuit.rs

```diff
@@ -242,3 +249,74 @@ impl TargetCircuit for PoseidonCircuit {
         hash_rows
     }
 }
+
+pub struct ByteCodeCircuit {}
+impl TargetCircuit for ByteCodeCircuit {
+    type Inner = ByteCodeCircuitImpl<Fr>;
+
+    fn name() -> String {
+        "ByteCode".to_string()
```

Comment:
small case to be consistent with others

Reply:
updated in [c93e757](https://github.com/scroll-tech/common-rs/pull/61/commits/c93e757d6c57a85dac5a45a693651a5f4d11809c)